src/lib/functions/registry.mjs

```diff
@@ -14,6 +14,7 @@ import { getPathInProject } from '../settings.mjs'
 import NetlifyFunction from './netlify-function.mjs'
 import runtimes from './runtimes/index.mjs'
 
+const DEFAULT_URL_EXPRESSION = /^\/.netlify\/(functions|builders)\/([^/]+).*/
```

Comment:
We should be refactoring this so that the regex is shared with `isFunction`

https://github.com/netlify/cli/blob/556d5456d750861ec1b00842ef88bcff6eef6c87/src/utils/proxy.mjs#L85-L91